dts/arm/nordic/nrf51822.dtsi

```diff
@@ -78,6 +78,7 @@
 			reg = <0x40003000 0x1000>;
 			interrupts = <3 NRF_DEFAULT_IRQ_PRIORITY>;
 			max-frequency = <DT_FREQ_M(8)>;
+			easydma-maxcnt-bits = <8>;
```

Comment:
These numbers should be in some way validated with what MDK provides in `nrf_peripherals.h`. I think something similar to `soc/arm/nordic_nrf/validate_base_addresses.c` is needed for this.

Reply:
I think we should actually get rid of validate_base_addresses.c. Nordic could easily provide verified DTS files, or, have unit tests that validate this against a "trusted source" like MDK. But from a Zephyr perspective, I think DT should be the source of truth, not MDK files.